Containerfile

```
@@ -0,0 +1,8 @@
+FROM quay.io/operate-first/opf-toolbox:v0.4.2 AS toolbox
+FROM quay.io/thoth-station/s2i-custom-py38-notebook:v0.3.2
+
+ENV XDG_CONFIG_HOME=/usr/share/.config
+ENV KUSTOMIZE_PLUGIN_PATH=$XDG_CONFIG_HOME/kustomize/plugin/
+
+COPY --from=toolbox /usr/local/bin/* /usr/local/bin/
```

We want all content of the `/usr/local/bin` in case there's something nested in subdirectories the command above won't copy it over. `abc/*` is not recursive. We want to copy the whole directory as is here:

```suggestion
COPY --from=toolbox /usr/local/bin /usr/local/bin
```

Your command is also much slower and demanding, since it has to do file listing in the overlay, figure out which chunks of the overlay to copy etc. instead of doing it only once

---

Resolved in latest commit.